docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -40,6 +43,8 @@ See the appropriate section for your hypervisor and operating system:
 # chmod +x /usr/local/bin/docker-machine-driver-kvm
 ----
 
+Proceed to the instructions for your Linux distribution after you have set up the docker-machine-driver-kvm binary.
```

Comment:
Nitpick: `docker-machine-driver-kvm` should be styled monospace. :)

Reply:
@robin-owen thanks for pointing it out, I had missed it :+1: